client/asset/eth/eth.go

```diff
@@ -408,16 +412,22 @@ type TokenWallet struct {
 	token    *dexeth.Token
 }
 
+const maxProportionOfBlockGasLimitToUse = 4
```

Comment:
ACK that 25% of a block's gas limit is a reasonable default.  Just please comment here to explain.